arwen/host/execution_test_vmoutputs.go

```diff
@@ -888,7 +888,7 @@ func expectedVMOutput_AsyncCall_CallBackFails() *vmcommon.VMOutput {
 	AddFinishData(vmOutput, []byte{3})
 	AddFinishData(vmOutput, []byte("thirdparty"))
 	AddFinishData(vmOutput, []byte("vault"))
-	AddFinishData(vmOutput, []byte("execution failed"))
+	AddFinishData(vmOutput, []byte("user error"))
```

Comment:
Yes, you get the specific error from the async called function in the callback, instead of the general "execution failed". The error message can be used in the callback.